src/tests/system/tests/test_cache.py

```diff
@@ -412,3 +414,83 @@ def test_cache__both_ldap_user_email_and_extra_attribute_email_are_stored(client
     assert user_dict["description"] == ["gecos1"], "attribute 'description' was not correct"
     assert user_dict["mail"] == ["user1@example.test"], "attribute 'mail' was not correct"
     assert user_dict["email"] == ["user1@example.test"], "attribute 'email' was not correct"
+
+
+@pytest.mark.parametrize(
+    "type, name",
+    [("group", "grp_nowait"), ("netgroup", "netgrp_nowait")],
+)
+@pytest.mark.importance("low")
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+def test_netgroup__sssd_entry_cache_nowait_percentage(client: Client, provider: GenericProvider, type: str, name: str):
+    """
+    :title: Test SSSD group caching with 'entry_cache_nowait_percentage'
```

Comment:
Can you please reference other tests and make the docstring similar? There is a lot of extra verbiage and it's quite hard to consume.